src/OrbitDB.js

```diff
@@ -295,10 +305,16 @@ class OrbitDB {
     // Create the database address
     const dbAddress = await this._determineAddress(name, type, options)
 
-    options.cache = this.caches[options.directory || 'default']
     if (!options.cache) {
-      const cacheStorage = await this.storage.createStore(options.directory)
-      this.caches[options.directory] = options.cache = new Cache(cacheStorage)
+      if (options.directory) {
+        const cacheStorage = await this.storage.createStore(options.directory)
+        this.caches[options.directory] = options.cache = new Cache(cacheStorage)
```

Comment:
Can this be broken into multiple lines instead of chaining? I think that would make it more readable and less error prone.

Reply:
:+1: